components/alibi-explain-server/pyproject.toml

```diff
@@ -16,9 +15,8 @@ xgboost = "1.4.2"
 catboost = "0.24.4"
 tornado = "^6.1"
 fastapi = "0.68.1"
-
-shap = { git = "https://github.com/slundberg/shap.git", tag = "v0.39.0" }
 protobuf = "^3.17.3"
+alibi = {extras = ["shap"], version = "^0.6.2"}
```

Comment:
Let's keep source of dependency as Github (but bump to 0.40.x indeed) as discussed offline

Reply:
Done